.github/workflows/ci.yml

```diff
@@ -44,6 +42,38 @@ jobs:
         env:
           TEST_FASTTEXT: true
       - uses: codecov/codecov-action@v3
+  check-without-extras:
+    name: Check cleanlab runs without optional dependencies
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - name: Install dependencies
+        run: |
+          python -m  pip install --upgrade pip
+          pip install .
+          pip install pytest pytest-lazy-fixture
+      - name: Run tests
+        run: | # these tests provide 50% coverage
+          pytest tests/test_multilabel_classification.py tests/test_multiannotator.py tests/test_filter_count.py
+  check-min-version:
+    name: Checks the package is compatible with minimum versions
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.7'
+      - name: Install min version of dependencies
+        run: |
+          python -m  pip install --upgrade pip
+          pip install .
+          pip install pytest pytest-lazy-fixture numpy==1.20.0 scikit-learn==1.0 tqdm==4.53.0 pandas==1.1.5 termcolor==2.0.0
+      - name: Run tests
+        run: | # this test provides 33% coverage
+          pytest tests/test_classification.py
```

Comment:
why not run the same 3 unit tests as above for this check?

Reply:
Combined the two checks